src/@types/location.ts

```diff
@@ -50,9 +51,29 @@ export interface ILocationContent extends IContent {
     geo_uri: string;
     [LOCATION_EVENT_TYPE.name]: {
         uri: string;
-        description: string;
+        description?: string;
     };
     [TEXT_NODE_TYPE.name]: string;
-    [TIMESTAMP_NODE_TYPE.name]: string;
+    [TIMESTAMP_NODE_TYPE.name]: number;
 }
 /* eslint-enable camelcase */
+
+export function makeLocationContent(
+    text: string,
+    uri: string,
+    ts: number,
+    description?: string,
+): ILocationContent {
+    return {
+        "body": text,
+        "msgtype": MsgType.Location,
+        "geo_uri": uri,
+        [LOCATION_EVENT_TYPE.name]: {
+            uri,
+            description,
+        },
+        [TIMESTAMP_NODE_TYPE.name]: ts,
+        [TEXT_NODE_TYPE.name]: text,
+        // TODO: MSC1767 fallbacks m.image thumbnail
+    };
+}
```

Comment:
feels a bit strange to have utility methods inside `/@types/`

Reply:
Where would you suggest?

Reply:
`content-helpers.ts`